config-connector/solutions/iam/kpt/pubsub-topic/README.md

```diff
@@ -0,0 +1,56 @@
+Pubsub Topic
+==================================================
+
+# NAME
+
+  pubsub-topic
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to grant a role to a particular IAM member for a PubSub topic.
+
+# CONSUMPTION
+
+  Using [kpt](https://googlecontainertools.github.io/kpt/):
```

Comment:
Let's use the text before the command to describe the command. For example, here I'd say something like "Fetch the [kpt](https://googlecontainertools.github.io/kpt/) package of the solution".